app/Http/Controllers/Admin/AdminHotelController.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace HotelBooking\Http\Controllers\Admin;
+
+use Hash;
+use DB;
+use Session;
+use HotelBooking\Hotel;
+use HotelBooking\AdminHotel;
+use Illuminate\Http\Request;
+use HotelBooking\Http\Requests;
+use HotelBooking\Http\Requests\Admin\AdminHotelCreateFormRequest;
+use HotelBooking\Http\Requests\Admin\AdminHotelUpdateFormRequest;
+use HotelBooking\Http\Controllers\Controller;
+use HotelBooking\Http\Controllers\Admin\AdminBaseController;
+
+class AdminHotelController extends AdminBaseController
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        $hotels = DB::table('hotels')
+            ->lists('name', 'id');
+        return view('admin.admin-hotel.create', compact('hotels'));
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request\Admin\
+     *          AdminHotelCreateFormRequest  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(AdminHotelCreateFormRequest $request)
+    {
+
+        $data = $request->all();
+        if (AdminHotel::create($data)) {
+            Session::flash('flash_message', trans('messages.create_success'));
```

Comment:
The key is `create_success` but the content is `You had successfully created an account`. What happen if you want to add new message when create another entity like `hotel`, `room_types` ....